src/components/super-networked-interactable.js

```diff
@@ -71,7 +71,6 @@ AFRAME.registerComponent("super-networked-interactable", {
         this.el.setAttribute("body", { type: "dynamic" });
         this._syncCounterRegistration();
       } else {
-        this.el.emit("grab-end", { hand: this.hand });
```

Comment:
mind explaining this change?

Reply:
These changes uncovered an existing issue with pinning, where grab-end would be fired when a remote user took over ownership of a pinned entity. It would cause all other users to re-pin an entity before the new owner un-pinned it. This was especially a problem now that pinning is gated.

Removing this emit was a bad solution though. I've gone with a different solution with Kevin's advice: https://github.com/MozillaReality/networked-aframe/pull/20